.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,18 @@
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+name: Property Testing
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  property-testing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
```

Comment:
We could ignore the submodule, we now use stacks-core directly from the repo.

Reply:
Done.